geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -187,27 +193,49 @@ The header can be in a separate file, diff in the `*.html.ejs` file:
 
 The content will be in the `geoportal/{{package}}_geoportal/static/header.html` file.
 
+Print
+-----
 
-Information
+- With the default configuration we can easier switch from local print and mutualised print
+- A new attribute is added and containy the client timezone
+- If you use the most recent version of the `legend.jrxml` print template, you will obtain a legend with
+  elements indented hierarchically per groups level. If you want to keep to previous legend style, keep your
+  previous `legend.jrxml` template and add a new variable `gmfPrintOptions.legend.showGroupsTitle` in the
+  `interfaces_config` constants of the vars file and set it to false.
+
+Editing
+-------
+
+- The default order column for edition enumeration lists is now the value column ("name" per
```

Comment:
this confuses me, the value column is "name" per default?

Reply:
Yes...